src/main/java/lotto/domain/LottoMachine.java

```diff
@@ -1,21 +1,97 @@
 package lotto.domain;
 
+import lotto.exception.LottoPurchaseAmountException;
+
+import java.math.BigDecimal;
 import java.util.ArrayList;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import static lotto.domain.Number.MAX_NUMBER;
+import static lotto.domain.Number.MIN_NUMBER;
+import static lotto.utils.RandomNumberUtils.generateRandomNumbers;
 
 public class LottoMachine {
 
+    public static final int LOTTO_SIZE = 6;
+    public static final Money GAME_PRICE = new Money(BigDecimal.valueOf(1000));
+
     public List<LottoNumber> getLottoList(Money money) {
-        return generateLotto(money.getPurchaseCount());
+        validationMoney(money);
+        int autoCount = getPurchaseCount(money, GAME_PRICE);
+        return generateAutoLotto(autoCount).stream()
+                .collect(Collectors.toList());
```

Comment:
stream을 사용하지 않아도 될 것 같아요 :)
```suggestion
        return new ArrayList<>(generateAutoLotto(autoCount));
```